subsys/fb/cfb.c

```diff
@@ -150,6 +150,41 @@ int cfb_print(const struct device *dev, char *str, uint16_t x, uint16_t y)
 	return -1;
 }
 
+int cfb_invert_area(const struct device *dev, uint16_t x, uint16_t y,
+		    uint16_t width, uint16_t height)
+{
+	const struct char_framebuffer *fb = &char_fb;
+
+	if ((fb->screen_info & SCREEN_INFO_MONO_VTILED) && !(y % 8)) {
+		if ((x >= fb->x_res) || (y >= fb->y_res)) {
+			LOG_ERR("Coordinates outside of framebuffer");
+
+			return -1;
+		}
```

Comment:
That is independent of `fb->screen_info`, you can move it up.
Please `return -EINVAL;`, you can fix existing `return -1;` in appropriate places to be consistent, but if please do it in separate commit.

Reply:
Return values replace in all places, added as preceding commit. :+1: